### ISSUE TITLE
Should not update "auto_updater" after releasing the app

### ISSUE THREAD

**Issue by maxclaus** [jenslind/electron-release#14]
Updating that file after releasing the app ([here](https://github.com/jenslind/electron-release/blob/master/src/cli.js#L36)) means the app was published with the wrong file.
I mean if I'm publishing a new version of my app. The "auto_updater" file in the git tag will include the previous version.

I think maybe a better solution would be check if the "auto_updater" file is updated before publishing it. Instead of updating that file with the latest version.

**Comment by maxclaus** [comment]
By the way, I could change it. Just would like to confirm it before writing anything down.

**Comment by jenslind** [comment]
The url to the new released `.zip` is returned when the release is done.
https://github.com/jenslind/electron-release/blob/master/src/index.js#L84

And it's that url that is used to update the `auto_updater.json`.
https://github.com/jenslind/electron-release/blob/master/src/cli.js#L36

**Comment by maxclaus** [comment]
Yeah. Thats is the problem. It is creating a tag in git before updating the `auto_updater.json` file.
If I access that tag it will include in the `auto_updater.json` the url to the previous version.

**Comment by maxclaus** [comment]
I will not implement the auto update in my app right now. Because the enrollment for the Apple Developer Program is too expensive. So if you prefer close this issue it is ok for me.

But I still think my suggestion would be a better approach :smile:

**Comment by akgpt** [comment]
i am not able to auto update my app. I used azure blob storage to store my app files (.exe, release, .nupkg). and provided container path as feedURL to autoUpdater. but my app is not auto updating. Do any one have solution for this. If yes then kindly help. Stuck on this for long time
